data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/exception/LogstashParsingException.java

```diff
@@ -0,0 +1,14 @@
+package org.opensearch.dataprepper.logstash.exception;
+
+/**
+ * Exception for visitor when it's unable to convert into Logstash models
```

Comment:
I'd like the javadoc to be more descriptive about what is Visitor? You can update this to `LogstashVisitor` and in that mention what the visitor class does.